bitnami/mongodb/Chart.yaml

```
@@ -26,4 +26,4 @@ name: mongodb
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/mongodb
   - https://mongodb.org
-version: 13.6.7
+version: 13.7.7
```

This is a new feature, so it would be a minor
```suggestion
version: 13.7.0
```